homework/calculate/calculate.hpp

```diff
@@ -2,6 +2,44 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
+    int r;
```

Comment:
What is 'r'? Variable should always represent real operation. Like `result` `divider` `multiplier` `error_code` etc...